packages/ui/forms/verify-identity/index.tsx

```diff
@@ -325,7 +338,7 @@ class VerifyIdentityForm extends Component <Props, State>{
     }
 
     render() {
-        const vertOffset = Platform.OS === 'android' ? -900 : -600
+        const vertOffset = Platform.OS === 'android' ? -600 : -300
```

Comment:
This needs to be -300 for Android (I think) and certainly 0 for iOS, a negative number on iOS causes this not to function properly.